cos-fleetshard-operator-camel/src/main/java/org/bf2/cos/fleetshard/operator/camel/CamelOperandConfiguration.java

```diff
@@ -63,7 +63,13 @@ interface Health {
     }
 
     interface ExchangePooling {
-        @WithDefault("pooled")
+        // The pooled exchange factory strategy has side effects on some components:
+        // - https://issues.redhat.com/browse/MGDCTRS-1058
+        // - https://issues.redhat.com/browse/MGDCTRS-1057
+        // - https://issues.apache.org/jira/browse/CAMEL-18187
+        //
+        // Default changed to "prototype" till we get a fix in Apache Camel
+        @WithDefault("prototype")
```

Comment:
changed the default instead of using a config option because given the issue, `prototype` seems to be a sensible default for now

Reply:
👍